app/src/main/java/fr/free/nrw/commons/nearby/NearbyFragment.java

```diff
@@ -813,6 +813,19 @@ public void onPause() {
         }
     }
 
+                      
+    /**
+     * Centers the map in nearby fragment to a given place
+     * @param place is new center of the map
+     */
+    public void centerMapToPlace(Place place) {
+        if (nearbyMapFragment == null) {
+            nearbyMapFragment.nearbyplace = place;
```

Comment:
Hi @sp2710 , this line would crash with NPE if the nearbyMapFragment is null. Can you explain what did you aim to achieve here?